packages/hunspell.rb

```diff
@@ -12,27 +13,24 @@ class Hunspell < Package
   is_fake
 
   if ARGV[0] == 'install'
-    puts
-    puts 'Enter your preferred language:'
-    puts '1 = American English'
-    puts '2 = Français'
-    puts '3 = Español'
-    puts '0 = Cancel'
-
-    while version = $stdin.gets.chomp
-      case version
-      when '1'
-        depends_on 'hunspell_en_us'
-        break
-      when '2'
-        depends_on 'hunspell_fr_fr'
-        break
-      when '3'
-        depends_on 'hunspell_es_any'
-        break
-      when '0'
-        abort
+    begin
+      @timeout = 10
+      @user_input = Timeout.timeout(@timeout) do
+        puts
+        puts 'Enter your preferred language:'
+        puts "(The default, American English, will be selected in #{@timeout} seconds if there is no input.)"
```

Comment:
Can `@timeout` display a countdown in real time?  If that can be done then we don't need `if there is no input`?

Reply:
No, as we can't get the current countdown from `Timeout`

Reply:
Alternate code to do that would be welcomed!

Reply:
I think the issue is accepting input while the countdown timer is running.